examples/platform/silabs/BaseApplication.cpp

```diff
@@ -175,25 +179,32 @@ BaseApplicationDelegate BaseApplication::sAppDelegate = BaseApplicationDelegate(
 void BaseApplicationDelegate::OnCommissioningSessionStarted()
 {
     isComissioningStarted = true;
+
+#if SL_WIFI && CHIP_CONFIG_ENABLE_ICD_SERVER
```

Comment:
```suggestion
#if SL_WIFI && defined(CHIP_CONFIG_ENABLE_ICD_SERVER) && CHIP_CONFIG_ENABLE_ICD_SERVER
```

Reply:
`CHIP_CONFIG_ENABLE_ICD_SERVER` is always defined so we don't need the extra define check.